### ISSUE TITLE
Inconsistent bolding in Chinese - Teacher Registration

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1126]
Staging, Firefox 50.0.2, Mac OS El Capitan

**TL;DR** I think in some places the Chinese characters are bolded, sometimes they aren't - I'm not sure if that is how they should look?

If you choose Chinese (the second to last option in the language dropdown menu on the homepage), then go to Teacher Registration, it looks like the bolding may be inconsistent. I'm not totally sure if that is true - it might be good to have someone who reads Chinese verify whether that is what's going on, or if it's correct as-is.

<img width="1181" alt="screen shot 2016-12-16 at 9 45 10 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283909/9c8ae3be-c3da-11e6-8205-e344cce3cabb.png">
<img width="305" alt="screen shot 2016-12-16 at 9 42 53 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283911/9c8cdd2c-c3da-11e6-9394-edb3f14d9038.png">
<img width="310" alt="screen shot 2016-12-16 at 9 42 44 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283910/9c8b4c8c-c3da-11e6-987c-a135d472131c.png">
<img width="505" alt="screen shot 2016-12-16 at 9 41 36 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283912/9c8d318c-c3da-11e6-9874-cc11e3b1c233.png">

**Comment by jwzimmer** [comment]
I'm really not sure if I'm able to tell whether the Chinese chars are bolded or not... Here's the username step in Chinese on Production in Chrome 55:
<img width="352" alt="screen shot 2016-12-16 at 10 03 02 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283960/9023fc72-c3db-11e6-93f3-a1b6bbdee8f0.png">

And on Firefox 50.0.2 on Production:
<img width="338" alt="screen shot 2016-12-16 at 10 05 46 pm" src="https://cloud.githubusercontent.com/assets/1320456/21283980/ef59f106-c3db-11e6-9703-cf513d8c7f2a.png">

To me, the 1st char looks bolded in Chrome but not Firefox, but I could just be totally wrong...

**Comment by Kenny2github** [comment]
As a person who does, in fact, read Chinese, I can assure you that the bolding is indeed inconsistent.

In some default Chinese fonts, some characters seem more bold than others - partially because of the stroke density, and partially because of the size of the character itself.
Characters like 密 (the first character in the word for "password"), with their higher stroke density (i.e. higher stroke count to same character dimensions), may seem less bold than characters like 码 (the second character in the word for "password").
This is especially more pronounced when you try to make the characters bold - some characters are too dense to be further bolded, while others are correctly bolded.

Consider this screenshot:
![chi](https://user-images.githubusercontent.com/28599280/31075171-cb127e44-a7a6-11e7-8bac-1301efea159d.png)
On top is bold, on the bottom is not. Compare it with your Firefox screenshot:
![sc](https://cloud.githubusercontent.com/assets/1320456/21283980/ef59f106-c3db-11e6-9703-cf513d8c7f2a.png)
码 seems bolder than 密, but they're both supposed to be bold, as per your Chrome screenshot:
![sc](https://cloud.githubusercontent.com/assets/1320456/21283960/9023fc72-c3db-11e6-93f3-a1b6bbdee8f0.png)
In conclusion, the bolding is there, however font issues make it so that some characters appear not bold. This issue also appears to be browser-specific _and_ font-specific - when it appears depends on the font, and according to your screenshots it only appears on Firefox.
